pkg/cmd/server/start/start_master.go

```diff
@@ -306,11 +306,13 @@ func StartMaster(openshiftMasterConfig *configapi.MasterConfig) error {
 
 	unprotectedInstallers := []origin.APIInstaller{}
 
-	authConfig, err := origin.BuildAuthConfig(*openshiftMasterConfig)
-	if err != nil {
-		return err
+	if openshiftMasterConfig.OAuthConfig != nil {
```

Comment:
Not blocking this pull, but if this is a scenario we really want to support, should we have an integration test for it?  Do we have any idea how to test success?